doc/source/index.md

```diff
@@ -134,6 +134,8 @@ advanced_examples/index.rst
 :maxdepth: 2
 
 api
+authors
+license
 config
```

Comment:
My only comment would be to move the Authors + License pages at the very end of the list here. (to have Configuration/Release notes pages showing before, as it seems more important for users)

We could also think about merging them together and potentially into sub-sections of the "Background" page instead of each being their own page, or into a new section "Project information" that includes "Publications", "License", "Authors", etc... (to avoid mixing these with the API), but that can be discussed/done later! :slightly_smiling_face:

Reply:
@rhugonnet  for now I added a new section "Project information" at the end of the documentation, that includes "Publications", "Background", "Authors" and "License".

Reply:
Perfect!